src/components/Details/common/tests/details.js

```diff
@@ -58,6 +59,7 @@ export const disablesEditOnCancel = component =>
   awaitVmDetails(() => {
     expect(component.find(InlineFormFactory).exists()).toBeFalsy();
     clickButton(component, 'Edit');
+    component.setState({ updating: false });
```

Comment:
is this because of the waiting on retrieveVmTemplate? Can we rather mock it and wait for the promise to finish?

Reply:
> Can we rather mock it and wait for the promise to finish?

I can, It's already mocked, I tried to wait for the promise and then removed the `async` and `await flushPromises` because I couldn't make it look nice, I can try again, maybe re-factor it this time to look nicer :-)